tests/TestCIT_KCI.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+
+import numpy as np
+
+import causallearn.utils.cit as cit
+
+class TestCIT_KCI(unittest.TestCase):
+    def test_Gaussian(self):
+        X = np.random.randn(300, 1)
+        X1 = np.random.randn(300, 1)
+        Y = np.concatenate((X, X), axis=1) + 0.5 * np.random.randn(300, 2)
+        Z = Y + 0.5 * np.random.randn(300, 2)
+
+        cit_CIT = cit.CIT(data=np.hstack((X,X1,Y,Z)), method='kci')
+
+        pvalue = cit_CIT.kci(0, 1, {})
```

Comment:
`cit_CIT.kci(0, 1, {})`: `{}` is a dictionary, not a set (I guess what you want here is `set()`?)

Reply:
Fixed, thanks.